src/python_ms_core/core/auth/models/permission_request.py

```diff
@@ -51,7 +51,7 @@ def get_search_params(self):
         affirmative = 'true' if self._should_satisfy_all else 'false'
         params = {
             'userId': self._user_id,
-            'agencyId': self._org_id,
+            'agencyId': self.project_group_id,
```

Comment:
Is this the same parameter `agencyId` ?

Reply:
AgencyId also to be replaced with projectGroupId